docs/concepts/function-components.md

```diff
@@ -52,3 +59,8 @@ Yew comes with the following predefined Hooks:
 
 There are cases where you want to define your own Hooks for reasons. Yew allows you to define your own Hooks which lets you extract your potentially stateful logic from the component into reusable functions. 
 See the [Defining custom hooks](function-components/custom-hooks.md#defining-custom-hooks) section for more information.
+
+## Further reading
+
+* The React documentation has a section on [React hooks](https://reactjs.org/docs/hooks-intro.html).
+These are not exactly the same as Yew's hooks, but the underlying concept is similar.
```

Comment:
Should we really recommend consulting the React docs? Maybe this could just be a link for people already familiar with React and wishing to try out Yew.